infra/gcp/main.tf

```diff
@@ -35,6 +35,7 @@ module "gcp_jumpbox" {
   vpc_subnet              = module.gcp_base[0].vpc_subnets[0]
   tsb_version             = var.tsb_version
   jumpbox_username        = var.jumpbox_username
+  machine_type            = var.jumpbox_machine_type
```

Comment:
Exposing it because the current default is not available in newer regions such as europe-southwest.